### DISCUSSION TITLE
Diagnostics for `CoulombCollisions`

### DISCUSSION THREAD

**Discussion by Sbozzolo** [BLAST-WarpX/warpx#4314] [Q&A]
Hello, I am new to WarpX. 

I got some simulations going, and now I am interested in adding Coulomb collisions for non my non-background species. To do so, I roughly followed an [example](https://github.com/ECP-WarpX/WarpX/blob/development/Examples/Tests/collision/PICMI_inputs_2d.py)  in the tests and added a `CoulombCollisions` to my simulation. However, I don't see much difference in my runs with respect to the case without collisions. 

I was wondering if there were ways to obtain diagnostics about `CoulombCollisions` to help me understand what is going on.

Thanks in advance.

**Comment by ax3l** [comment]
Hi @Sbozzolo, we do not have a diagnostics about Coulomb collisions per se, but you can diagnose (output) the particle momentum (and position, etc.) at any time step.

I assume that in your simulation either the coulomb log is not correctly set of the mean collision frequency for the densities at play is very long and thus your plasma/beam is not significantly influenced by collisional physics.

Do you have the respective references for collision physics and PIC models we implement already at hand or prefer some pointers to literature to check these quantities in your scenario?

**Comment by Sbozzolo** [comment]
Thanks, @ax3l. 

> I assume that in your simulation either the coulomb log is not correctly set of the mean collision frequency for the densities at play is very long and thus your plasma/beam is not significantly influenced by collisional physics.

I tried both not setting the Coulomb log, and setting it to a small value. How do you pick the correct value?

How does the grid resolution affect the Coulomb collisions?

> Do you have the respective references for collision physics and PIC models we implement already at hand or prefer some pointers to literature to check these quantities in your scenario?

Yes, it would be great to be pointed to some trusted sources.

Thanks!
